### PR TITLE
Test for checking the behavior of shutdown(SHUT_WR)

### PR DESCRIPTION
**Pull request by shivrai**
This test checks for the behavior of the sequence :
**shutdown(SHUT_RD)**, **read**, **shutdown(SHUT_WR)**, **close**.

### CODE REVIEW

File: gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+0.100 < S 0:0(0) win 65535 <mss 1460,sackOK,nop,nop,nop,wscale 7>
+0.100 > S. 0:0(0) ack 1 <...>
+0.200 < . 1:1(0) ack 1 win 65535
+0.200 accept(3, ..., ...) = 4
+
+0.210 shutdown(4, SHUT_RD) = 0
+
+// Verify if read works (It should not work)
+0.210 read(4, ..., 1000) = 0
```
Comment (tuexen) [inline gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt]: Again: Is this the appropriate handling?

File: gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+0.250 shutdown(4, SHUT_WR) = 0
+
+// Verify if write works (It should not work)
+0.260 write(4, ..., 1000) = -1 EPIPE (Broken pipe)
+
+0.260 > F. 1:1(0) ack 1
```
Comment (tuexen) [inline gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt]: I would expect this segment to be sent right after the `shutdown(4, SHUT_WR)` call is issued,
not 10ms later.

Comment (shivrai) [inline gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt]: Updated, thanks.

File: gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt (hunk 1 above)
Comment (tuexen) [inline gtests/net/packetdrill/tests/bsd/tcp/shutdown/shutdown-rd-wr.pkt]: If you take the code from line 15 to line 18 out, we agree on the behaviour and I would accept it.

**Pull request closed without merging**